assignments/prototype-refactor.js

```diff
@@ -7,3 +7,180 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+/*
+  Object oriented design is commonly used in video games.  For this part of the assignment you will be implementing several constructor functions with their correct inheritance hierarchy.
+
+  In this file you will be creating three constructor functions: GameObject, CharacterStats, Humanoid.  
+
+  At the bottom of this file are 3 objects that all end up inheriting from Humanoid.  Use the objects at the bottom of the page to test your constructor functions.
+  
+  Each constructor function has unique properties and methods that are defined in their block comments below:
+*/
+  
+/*
+  === GameObject ===
+  * createdAt
+  * name
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+class gameObject {
```

Comment:
This won't change the functionality of the code, but _by convention_, the first letter is capitalized for constructors/classes. You've done this in the other file, so just remember to be consistent.